mods/cnc/rules/world.yaml

```diff
@@ -6,6 +6,7 @@
 	MusicPlaylist:
 		VictoryMusic: win1
 		DefeatMusic: nod_map1
+	DebugVisualizations:
```

Comment:
Probably more trouble than its worth to stop it from adding these to maps that override World.